client/gutenberg/extensions/slideshow/style.scss

```diff
@@ -0,0 +1,68 @@
+/** @format */
+
+.wp-block-jetpack-slideshow {
+	position: relative;
+	.swiper-container {
+		width: 100%;
+		height: 100%;
+	}
+	img {
+		height: 100%;
+		height: calc( 100% - 22px );
+		background-color: rgb( 240, 240, 240 );
+		background-size: contain;
+		background-repeat: no-repeat;
+	}
+	img,
+	figcaption {
+		position: relative;
+		z-index: 1;
+	}
+	figcaption {
+		cursor: text;
+	}
+	.swiper-button-prev,
+	.swiper-button-next {
+		width: 24px;
+		height: 40px;
+		background-size: 24px 40px;
+		box-sizing: content-box;
+		-moz-box-sizing: content-box;
```

Comment:
I don't think we should be using CSS property prefixes, we have tooling that will add them as required.

Reply:
If I remove `-moz-box-sizing` but leave `box-sizing: content-box;`, I end up with the following in the webpacked css files:

```
-webkit-box-sizing: content-box;
            box-sizing: content-box;
```

It looks like the SDK tooling adds `-webkit-` prefix but not `-moz-`. So the question becomes, is this an oversight or do we not need `-moz` prefixing? For now I'm assuming we don't and removing the prefix in https://github.com/Automattic/wp-calypso/pull/29419/commits/cd8d204be845f254c03032c55d3eb8ed5a7038b5

Reply:
Let's go ahead and remove the prefixes. In the box-sizing case, `-moz` prefix was required up to Firefox 28 which was superseded in 2014 and has very small usage:

https://caniuse.com/#search=box-sizing

Our support for CSS is defined here and our tooling will handle prefixes. Provided we don't need to support FF 28 and below, it doesn't add `-moz` to `box-sizing`. It may well strip it from the output– although I haven't confirmed that. Less reason to keep it here in the source 🙂 

https://github.com/Automattic/wp-calypso/blob/9736e70bf0157c34ba1e02c97b22670b04622458/package.json#L15-L21

Reply:
OK, thanks for all the detail. The prefixes were removed in https://github.com/Automattic/wp-calypso/commit/cd8d204be845f254c03032c55d3eb8ed5a7038b5 so we should be all good on this.